integration_tests/tests/apply/complete_about_the_person_section.cy.ts

```diff
@@ -0,0 +1,192 @@
+//  Feature: Referrer completes 'About the person' section
+//    So that I can complete the 'About the person' section
+//    As a referrer
+//    I want to answer questions within that section of the task list
+//
+//  Scenario: Follows link from task list
+//    Given there is a section with a task
+//    And an application exists
+//    And I am logged in
+//    And I am viewing the application task list
+//
+//  Scenario: view task listed within the section
+//    Then I see the task listed within the section
+//    And I see that the task has not been started
+//
+//  Scenario: follow link to first task page
+//    When I follow the link within the section
+//    Then I'm on the expected task page
+//    And the task list page has the expected question and answers
+//
+//  Scenario: answer is enforced
+//    Given I'm on the 'Will answer equality questions?' task page
+//    When I try to continue without answering the question
+//    Then I see that an answer is required
+//
+//  Scenario: return to task list using the back button
+//    Given I'm on the 'Will answer equality questions?' task page
+//    When I use the back button
+//    Then I'm on the task list page
+//
+//  Scenario: task completed successfully
+//    Given I'm on the 'Will answer equality questions?' task page
+//    When I select an option and choose to save and continue
+//    Then I'm returned to the task list
+//    And I see that the task is now complete
+
+import Page from '../../pages/page'
+import TaskListPage from '../../pages/apply/taskListPage'
+import WillAnswerEqualityQuestionsPage from '../../pages/apply/willAnswerEqualityQuestionsPage'
+import { personFactory, applicationFactory } from '../../../server/testutils/factories/index'
+
+context('Visit area and funding section', () => {
+  const person = personFactory.build({ name: 'Roger Smith' })
+
+  beforeEach(function test() {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+
+    cy.fixture('applicationData.json').then(applicationData => {
+      const application = applicationFactory.build({
+        id: 'abc123',
+        data: {
+          'area-and-funding': {
```

Comment:
minor question: do we need to build with the `area-and-funding` data in the application, or can that be deleted?

Reply:
yes, we need to put the area-and-funding task into a complete state so that the equality and diversity task in the "Not yet started" state. If not our new task will be in `Cannot yet be started`

Reply:
ah that's interesting, I hadn't looked at the logic in the `taskListService` that makes a section `cannot_start` - we were hoping to allow the user to start any task in any order (after the first section has been completed) so sounds like we need a user story to allow that to work.